open-apparel-registry/open-apparel-registry
# Add disambiguation popup for multiple facilities

## Overview

Implement a disambiguation popup which is displayed when a marker from a
set of search results has multiple facilities at the same point.

Connects #187 

## Demo

![facilities-popup](https://user-images.githubusercontent.com/4165523/53372807-cbf44c80-3921-11e9-97d2-d105e6fd1667.gif)

## Notes

~I still need to set up some viable test data for this, so I'll push another commit to do so~ I updated the `resetdb` command to do this at the end.

## Testing Instructions

- get this branch, then run `./scripts/manage resetdb`
- search for facilities, then click on the facilities at `Point(0, 0)` and verify that it zooms in and displays the popup as depicted in the demo gif
- selecting a facility from the list should link you to its details page
- verify that you can toggle through each of the multiple facilities and be linked to their details page
- verify that you can dismiss and re-create the marker
- verify that the markers for single facilities continue to work as before
